src/ng/compile.js

```diff
@@ -1698,62 +1698,64 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
 
   /**
    * @ngdoc method
-   * @name $compileProvider#aHrefSanitizationWhitelist
+   * @name $compileProvider#aHrefSanitizationTrustedUri
```

Comment:
FWIW, I would consider calling these `aHrefSanitizationTrustedUrlList` (and `imgSrcSanitizationTrustedUrlList` respectively) to match $sceDelegateProvider#trustedResourceUrlList`.

This will keep (some of) the `$compileProvider#aHrefSanitizationWhitelist`/`$sceDelegateProvider#resourceUrlWhitelist` resemblance.